templates/heading.hbs

```diff
@@ -1,19 +1,21 @@
 {{import_globals}}
 
 <div id="{{_id}}-heading" class="js-heading-inner" role="heading" aria-level="{{a11y_aria_level _id _type _ariaLevel}}">
-
   <span class="aria-label">
   {{#if _isA11yCompletionDescriptionEnabled}}
     {{#if _isOptional}}
-      {{{compile title}}}
       {{else if _isComplete}}
-      {{_globals._accessibility._ariaLabels.complete}}. {{{compile title}}}
+      {{_globals._accessibility._ariaLabels.complete}}.&nbsp;
```

Comment:
Is this &nbsp; coming from your editor?

Reply:
It's there because a space has to exist between the `Complete.` and `{{compile displayTitle}}`. Otherwise NVDA will read the period aloud, "CompleteDOTTitle" rather than "Complete[Pause]Title".